test_util/marathon.py

```diff
@@ -131,6 +132,11 @@ def deploy_test_app_and_check(self, app, test_uuid):
 
             assert r.json() == {'username': marathon_user}
 
+    def get_apps_tasks_details(self):
+        apps_details = self.get("v2/apps").json()
+        tasks_details = self.get("v2/tasks").json()
+        return json.dumps({"apps_details": apps_details, "tasks_details": tasks_details})
```

Comment:
Why is this returning strings? I feel like those two API calls should just be made when they are needed rather than taking generic API responses and rewrapping them into a customized dict that gets stringified